src/components/routeComponents/memberRegistrationCorrectionForm/classRoomApi.js

```diff
@@ -10,3 +10,14 @@ export const fetchClassRoomsByLevel = async ({ level, secretKey }) => {
 		},
 	});
 };
+
+export const fetchCoordinator = async ({ secretKey }) => {
+	return await GET({
+		url: `/v1/coordinators `,
```

Comment:
Please confirm the url param name with the backend team
```suggestion
		url: `/v1/coordinators/?whatsappNumber=${whatsappNumber} `,
```